frontend/src/routes/+layout.server.ts

```diff
@@ -1,10 +1,12 @@
 import type { LayoutServerLoadEvent } from './$types'
 import { getRootTraceparent } from '$lib/otel/server'
 
-export function load({ locals }: LayoutServerLoadEvent) {
+export function load({ locals, depends }: LayoutServerLoadEvent) {
   const user = locals.getUser();
   const traceParent = getRootTraceparent()
 
+  if (user) depends(`user:${user.id}`);
```

Comment:
But invalidating still won't have any effect until we implement the cookie getting updated right?

Reply:
Yes, at the moment this will just re-read the cookie. But once the `getUser` function is changed to actually load the user data from GraphQL, then the invalidate will start doing something.